### DISCUSSION TITLE
tool/google/request

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#149] [Announcements]
# tool/google/request

import Request from 

https://alist.nn.ci/tool/google/request.html

**Comment by meowmeow123221** [comment]
sorry this page dont have code blank can fill in .. ?
i see a previous tutorial video 
that page have code blank

**Comment by KittyPeace** [comment]
...

**Comment by dudupoppy** [comment]
禁止访问：此应用的请求无效


无法登录，因为此应用发送的请求无效。您可稍后再试，也可向开发者求助。
如果您是此应用的开发者，请参阅错误详情。
错误 400： redirect_uri_mismatch


人麻了 
导不进去
